lesson-07/task-07.js

```diff
@@ -0,0 +1,131 @@
+// =========================================
+// Task-01
+// =========================================
+
+const catigories = document.getElementsByTagName("h2");
+
+let output = (catigories) => {
+  Array.from(catigories).forEach((item) => {
+    console.log(
+      `Categories: ${item.textContent} </b> Quantity: ${item.nextElementSibling.children.length}`
+    );
+  });
+};
+
+output(catigories);
+
+// =========================================
+// Task-02
+// =========================================
+
+const ingredients = [
+  "Картошка",
+  "Грибы",
+  "Чеснок",
+  "Помидоры",
+  "Зелень",
+  "Приправы",
+];
+
+const ul = document.querySelector("#ingredients");
+
+ingredients.forEach((item) => {
+  let li = document.createElement("li");
+  li.textContent = item;
+  ul.append(li);
+});
+
+// =========================================
+// Task-03
+// =========================================
+
+const images = [
+  {
+    url:
+      "https://images.pexels.com/photos/140134/pexels-photo-140134.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260",
+    alt: "White and Black Long Fur Cat",
+  },
+  {
+    url:
+      "https://images.pexels.com/photos/213399/pexels-photo-213399.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260",
+    alt: "Orange and White Koi Fish Near Yellow Koi Fish",
+  },
+  {
+    url:
+      "https://images.pexels.com/photos/219943/pexels-photo-219943.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260",
+    alt: "Group of Horses Running",
+  },
+];
+
+let ul = document.querySelector("#gallery");
+
+ul.insertAdjacentHTML(
+  "afterbegin",
+  images.map((item) => {
+    let newImg = document.createElement("img");
+    newImg.src = item.url;
+    newImg.alt = item.alt;
+    return `<li> ${newImg.outerHTML}</li>`;
+  })
+);
+
+// =========================================
+// Task-04
+// =========================================
+
+let counterValue = 0;
+
+const incrementButton = document.querySelector(
+  "button[data-action='increment']"
```

Comment:
В JS прийнято використовувати одинарні лапки
```suggestion
  'button[data-action="increment"]'
```